operations/jsonnet/microservices/config.libsonnet

```diff
@@ -11,21 +11,71 @@
     gossip_member_label: 'tempo-gossip-member',
     compactor: {
       replicas: 1,
+      resources: {
+        requests: {
+          cpu: '500m',
+          memory: '3Gi',
+        },
+        limits: {
+          cpu: '1',
+          memory: '5Gi',
+        },
```

Comment:
Since the compactor is multi-threaded now, should we bump these resource settings?

Reply:
I don't think it really matters. None of these limits will work for a high scale deployment.